### PR TITLE
Error 404 page added and changes made

### PR DESCRIPTION
**Pull request by VC6378**
Heyy, 
Error 404 page has been added!!

### CODE REVIEW

File: package-lock.json
Comment (ShwetKhatri2001) [inline package-lock.json:1]: I guess this package-lock.json is creating a problem for netlify-deployment. 

Try this : 
- Delete `package-lock.json` file ( not package.json ) and `node_modules` folder
- Install the dependencies again using `npm install`, run the app using `npm start` and check if all your changes are working fine and then commit the changed file.

File: src/components/ModelViewer/Error404.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import imgerror  from '../ModelViewer/error404img.jpg';
+import '../ModelViewer/Error404_style.css';
+
+const Error404 = () => {
+
+    const redirectToHomePage = () => {
+        window.location.href = '/'; // Replace with the desired home page URL or path
+      };
+
```
Comment (ShwetKhatri2001) [inline src/components/ModelViewer/Error404.js:10]: Please change the above function and redirect using  useNavigate ( from react-router-dom )
Ref : https://stackoverflow.com/questions/34735580/how-to-do-a-redirect-to-another-route-with-react-router

**Review by ShwetKhatri2001 — CHANGES_REQUESTED** [review]
Make this small change and see why the deployment-preview is failing, try to fix that please.

**Comment by VC6378** [comment]
@ShwetKhatri2001 I tried deleting and doing the same, but it did not work. Do you have some other options?

**Comment by ShwetKhatri2001** [comment]
@VC6378 You can update your PR now. Pull the latest master branch. You shouldn't be getting any deployment-related errors. 
react-router is added there now. So, you just need to update this PR and make the small code change as I mentioned earlier: https://github.com/ShwetKhatri2001/AR-Webstore/pull/68#discussion_r1231364908

**Comment by ShwetKhatri2001** [comment]
Any updates or issues @VC6378  ?
Please try to complete the requested changes and update your PR within the next 3 days.
After that, This issue will be open to other interested contributors.

**Comment by ShwetKhatri2001** [comment]
Closing this PR because it has been inactive for a long time and the issue is resolved now.

**Pull request closed without merging**